scripts/auto-rebase/rebase_job_entrypoint.sh

```diff
@@ -27,7 +27,7 @@ release_arm64="$(oc image info --registry-config=/tmp/registry.json $OPENSHIFT_R
 #  can increment independently from LVMS, and will usually be 1 y-stream ahead of LVMS in-between OCP releases.
 #  For example, ocp-release at 4.13 will more often than not correspond to 4.12 LVMS, until the official 4.13 release when
 #  both components will be 4.13.
-release_lvms="v4.12"
+release_lvms="v4.13.1-5"
```

Comment:
Does this cause `last_rebase.sh` to be updated, too?

Reply:
when running `rebase_to` the last_rebase.sh will be updated with the latest manifest, however I seem to get stuck at a seemingly unrelated component: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_microshift/2107/pull-ci-openshift-microshift-main-test-rebase/1683554164476481536#1:build-log.txt%3A950

After this succeeds on the rebase, it should be updated in `last_rebase.sh` as well through `update_last_rebase` which is called in `rebase to`

Reply:
https://github.com/openshift/microshift/pull/2106 may help with that issue. You could try rebasing to pull in those changes.